.github/workflows/performance.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   # Runs the performance test suite.
   performance:
     name: Performance tests ${{ matrix.memcached && '(with memcached)' || '' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-performance.yml@trunk
+    uses: ./.github/workflows/reusable-performance.yml
```

Comment:
Curious why this needs to be updated? All the other workflows that rely on a reusable workflow use the previous pattern. It's probably best to keep them consistent unless it's necessary.

Reply:
@joemcgill Just posted an update in https://github.com/WordPress/wordpress-develop/pull/7490#issuecomment-2392097939, it's temporary so that the PR uses _this_ PR's workflow.

I'm using that as I make changes, but eventually it'll be changed back to `@trunk` before commit.

Reply:
I saw that after posting. Makes sense to do this temporarily 👍🏻